### ISSUE TITLE
The requested module (qs@6.11.1) does not provide an export named 'stringify'

### ISSUE THREAD

**Issue by Archiolidius** [nuxt-modules/strapi#331]
Any ideas on how to fix this issue?

```Uncaught SyntaxError: The requested module '/_nuxt/node_modules/.pnpm/qs@6.11.1/node_modules/qs/lib/index.js?v=0debfc0a' does not provide an export named 'stringify' (at useStrapiClient.mjs?v=0debfc0a:1:10)```

 I've tried different versions of "@nuxtjs/strapi" and "nuxt" but it doesn't help

"@nuxtjs/strapi": "^1.9.0"
"nuxt": "^3.3.1"
node v16.19.1

**Comment by ikluhsman** [comment]
I currently had this same issue with Nuxt 3.4.1, and Strapi/nuxt module version 1.9.1, and tailwindcss version 3.2.7. It turns out that after upgrading all packages to the latest versions (tailwind needed to upgrade to 3.3.1) everything started working again. From my experience, I did not find the root cause, but I hope this helps.

**Comment by benjamincanac** [comment]
Sorry for the late reply, do you still have the issue? Did not manage to reproduce myself.

**Comment by Smef** [comment]
I just installed this module to test it and am having this issue. The erroring line is `import { stringify } from "qs";`

Nuxt/strapi 2.0.0
Nuxt 3.16.2,

**Comment by khalidzamer** [comment]
Hello, I spent a full day wrestling with this issue, last night my app was running perfect, woke up and out of nowhere I get this error, I did not change anything and just one page stops being responsive and other pages are normal, like my signup page is busted my login page is fine, they are almost exact.

I also tried with GPT it was a total waste of time and sanity..

anybody have a solution please? I am sorry I am unable to send reproduction steps because I don't know where to start or pinpoint the issue

**Comment by fanvyr** [comment]
@khalidzamer @Smef yep, had the same right now as well.. for me it was the package-lock.json. 
to fix it:
- checkout an older commit where its working
- install all "later installed" npm packages, make sure its still working
- copy the generated package-lock.json & discard local changes
- checkout the original commit / branch and copy-over the package-lock file. 

fixed it for me.
